main.go

```diff
@@ -73,15 +72,10 @@ func main() {
 	}
 	logger.SetLevel(logLevel)
 
-	// Set the preferred portal address.
-	database.Portal = os.Getenv("PORTAL_DOMAIN")
-	if database.Portal == "" {
-		log.Fatal("missing env var PORTAL_DOMAIN")
-	}
-	if !strings.HasPrefix(database.Portal, "http") {
-		database.Portal = "https://" + database.Portal
-	}
 	// Set the unique name of this server.
+	//
+	// TODO: it might make sense to rename this to SERVER_NAME, SERVER_DOMAIN
```

Comment:
I wouldn't leave TODOs as suggestions like this since that just pollutes the code. If you care enough about this change we should create an issue and discuss it. If not then it's probably not going to get implemented and removed from the codebase ever.

Reply:
I'll remove the TODO for the time being. I don't mind opening up the discussion somewhere but I don't think it's important enough, given the current priorities, to be discussing this at this point in time.